src/WebUI/Pods/PodLog.tsx

```diff
@@ -10,8 +10,11 @@ import { KubeSummary } from "../Common/KubeSummary";
 import * as Resources from "../Resources";
 import { PodContentReader } from "./PodContentReader";
 import { IPodRightPanelProps } from "./PodsRightPanel";
+import { V1Pod } from "@kubernetes/client-node";
 
-export interface IPodLogProps extends IPodRightPanelProps { }
+export interface IPodLogProps extends IPodRightPanelProps {
+    pod: V1Pod;

```

Comment:
add this to IPodRightPanelProps

Reply:
The base class has this, but there I have to keep an option of this being undefined for refresh scenario, but these components should have it as not null

Reply:
would you add a comment, so we do not lose track of this fact please.